github/orgs_audit_log_test.go

```diff
@@ -34,6 +34,8 @@ func TestOrganizationService_GetAuditLog(t *testing.T) {
         "created_at": 1615077308538,
         "head_sha": "5acdeadbeef64d1a62388e901e5cdc9358644b37",
         "conclusion": "success",
+		"old_permission": "read",
+		"permission": "admin",
```

Comment:
Nit: Would you mind making these two lines use the same spacing as lines 34-36 above them?

Reply:
Just to clarify; you're referring to the use of spaces instead of tabs?

Reply:
Yes, exactly... sorry, and thank you!

Reply:
No worries at all; will do!